src/modules/updateTaskDescription.js

```diff
@@ -0,0 +1,10 @@
+import DataClass from './dataClass.js';
+import show from './show.js';
+// Method to update the task's description
+const updateTaskDescription = (index, description) => {
+  DataClass.updateTaskDescriptionData(description, index);
+  // Refresh the toDoList in the UI
+  show();
+};
```

Comment:
- [ ] It's great that you have implemented this code but I am only able to update a to-do description if I refresh the page after adding it. Please fix it :+1:

Reply:
Hello @HAMMAS-SALEEM , actually all the CRUD functionality was working before I sent it. But now it does not work, I think it could be due to problems with Webpack
![image](https://user-images.githubusercontent.com/53547956/179323652-bca47108-df5b-4982-af43-d9ab944eae22.png)
Could you help me?

Reply:
Sure reach out to me on slack @VaneCode so I guide you. Search my name **Hammas Bin Saleem** in the **full-time students** channel. :+1: